neo/Consensus/ConsensusService.cs

```diff
@@ -360,6 +411,20 @@ public static Props Props(NeoSystem system, Wallet wallet)
 
         private void RequestChangeView()
         {
+            if (context.State.HasFlag(ConsensusState.CommitSent))
```

Comment:
let's remove this and filter before, @shargon.

```cs 
else if ((context.State.HasFlag(ConsensusState.Primary) && context.State.HasFlag(ConsensusState.RequestSent)) || (context.State.HasFlag(ConsensusState.Backup) && !context.State.HasFlag(ConsensusState.CommitSent))    )
```

at line https://github.com/neo-project/neo/blob/6b23061695152ed03d4ff5f04ff87d1ed9affcef/neo/Consensus/ConsensusService.cs#L384